test/tests.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bats
+
+load test_helper/bats-support/load
```

Comment:
Wonder if we could change this to bring the dep in by cloning first - then we would not need to check in all of `bats-support` to the test folder